qDict/src/main/res/layout/about.xml

```diff
@@ -4,56 +4,73 @@
                 android:layout_width="match_parent"
                 android:layout_height="match_parent"
                 android:background="@color/mmt_white"
-                android:paddingBottom="15dp">
+                android:paddingBottom="15dp"
+                xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
Do we actually need to wrap it in ConstraintLayout? As far as I understand, you only need it to set relative width and you don't need a ConstraintLayout for this: https://stackoverflow.com/questions/4961355/percentage-width-in-a-relativelayout

Reply:
OK, I will check this.

Reply:
Hello, I just found out that `PercentRelativeLayout` is deprecated now, and the official developers suggest we should use `ConstraintLayout` instead, and you could refer to this one: https://github.com/JulienGenoud/android-percent-support-lib-sample

And I truly did some adjustments and I found I could use `LinearLayout` all the way with height written in percent format. However, the QDict string could not be set above the version string if I don't wrap them in another `LinearLayout`. So maybe `ConstraintLayout` is for the best. That is my opinion.

Sorry for the late, I am quite busy these days and I would like to reformat this code and commit it later.

Thank you so much!!

:)

Reply:
OK, I tried just now. (Sorry for misunderstanding your intention) And the code seems much tidier now. It seems good to me. Here is the screenshot:

![Screenshot_1630681351](https://user-images.githubusercontent.com/25502419/132026787-c3cd2664-f26a-46c8-9c10-de8993b86600.png)

Reply:
So, there are three nested `LinearLayout`s now.

Reply:
We're almost there, can you add some margin on the left of the icon?